src/database/common.c

```diff
@@ -398,7 +392,7 @@ int db_get_int(sqlite3* db, const enum ftl_table_props ID)
 
 	if(querystr == NULL || ret < 0)
 	{
-		logg("Memory allocation failed in db_get_int db, with ID = %u (%i)", ID, ret);
+		log_warn("Memory allocation failed in db_get_int db, with ID = %u (%i)", ID, ret);
```

Comment:
Memory issues are always pretty severe.

```suggestion
		log_err("Memory allocation failed in db_get_int db, with ID = %u (%i)", ID, ret);
```